cereus/src/main/java/com/cactus/adapters/WebGroceryAdapter.java

```diff
@@ -276,7 +269,7 @@ public GroceryList setGroceryItems(List<String> items, long listID, String token
         String responseBody = makeRequest(this.client, request, "Could not save to this list");
 
         try {
-            return new ObjectMapper()
+            new ObjectMapper()
```

Comment:
If we are not returning the GroceryList object anymore, then it might make more sense for efficiency to not even parse the response body anymore. Simply do the `makeRequest` call and finish.

Reply:
The same applies to other methods where the return has been removed.